cypress/component/BrDisplayUser.cy.tsx

```diff
@@ -0,0 +1,24 @@
+import React = require("react");
+import BrDisplayUser from "../../components/brickroom/BrDisplayUser";
+import "../../styles/globals.scss";
+
+describe("AddContributors.cy.tsx", () => {
+    it("something", () => {
```

Comment:
Leave a understandable description of the executed tests, something that a non programmer could understand (we have to show it and demonstrate the work done to a pletora of stakeholders)